### PR TITLE
[PW-3648] - Update README with supported payment methods

### PR DESCRIPTION
**Pull request by Morerice**
## Summary
Update README file with currently supported payment methods

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,39 @@
 Use Adyen's plugin for PrestaShop to offer frictionless payments online, in-app, and in-store.
 
 ## Integration
-The plugin integrates card component(Secured Fields) using Adyen Checkout for all card payments. Local payment methods are integrated with Checkout Payment Methods.
-
+The plugin integrates card component (Secured Fields) using Adyen Checkout for all card payments.
```
Comment (cyattilakiss) [inline README.md:5]: I think this can stay: "Local payment methods are integrated with Checkout Payment Methods."

File: README.md [hunk 2]
```diff
@@ -2,8 +2,39 @@
...
+- Paysafecard
+- POLi
+- Qiwi
+- RatePAY
+- SEPA Direct Debit
+- Swish
+- Trustly
+- TWINT
+- UnionPay
+- YooMoney
```
Comment (cyattilakiss) [inline README.md:37]:
```suggestion
- YooMoney

```

File: README.md [hunk 3]
```diff
@@ -2,8 +2,39 @@
...
 ## Integration
-The plugin integrates card component(Secured Fields) using Adyen Checkout for all card payments. Local payment methods are integrated with Checkout Payment Methods.
-
+The plugin integrates card component (Secured Fields) using Adyen Checkout for all card payments.
+
+### Supported payment methods
+- Credit and debit cards (non 3D secure, 3D secure 1, 3D secure 2 native)
+- Stored card payment methods (One click payment methods)
+- Alipay
+- Applepay
```
Comment (acampos1916) [inline README.md:11]:
```suggestion
- Apple Pay
```

**Review by acampos1916 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Morerice) [reply README.md]: https://github.com/Adyen/adyen-prestashop/pull/204/commits/93ed4c3d149d833b3312d51674d4de8d52816a81

File: README.md (hunk 2 above)
Comment (Morerice) [reply README.md:37]: https://github.com/Adyen/adyen-prestashop/pull/204/commits/93ed4c3d149d833b3312d51674d4de8d52816a81

File: README.md (hunk 3 above)
Comment (Morerice) [reply README.md]: https://github.com/Adyen/adyen-prestashop/pull/204/commits/93ed4c3d149d833b3312d51674d4de8d52816a81

**Review by cyattilakiss — APPROVED** [review]

**Review by acampos1916 — APPROVED** [review]

**Pull request merged**